debug/app/bespoke/smk-config.json

```diff
@@ -0,0 +1,141 @@
+{
+    "name": "Bespoke1",
+    "viewer": {
+        "type": "leaflet",
+        "location": {
+            "center": [
+                -124.97187974331459,
+                51.821895387033834
+            ],
+            "extent": [
+                -133.3683520649526,
+                46.021881710514336,
+                -116.57540742167657,
+                56.96082045105082
+            ],
+            "zoom": 5.662861104534402
+        },
+        "baseMap": "Streets"
+    },
+    "layers": [],
+    "tools": [
+        {
+            "type": "toolbar",
+            "enabled": true
+        },
+        {
+            "type": "list-menu",
```

Comment:
I think you can get rid of the list menu button since no tools are present in the list.